packages/authhero/src/types/Hooks.ts

```
@@ -21,6 +21,22 @@ export type OnExecuteCredentialsExchangeAPI = {
+    setUserMetadata: (key: string, value: any) => void;
+  };
+};
+
+export type OnExecutePreUserRegistration = (
+  event: HookEvent,
+  api: OnExecutePreUserRegistrationAPI,
+) => Promise<void>;
+
+export type OnExecutePostUserRegistration = (
+  event: HookEvent,
+  api: {},
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)